.github/workflows/release_master_fontpatcher-zip.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release font patcher from master
+
+on:
+  create:
+  push:
+    branches: master
+    paths:
+      - font-patcher
+      - src/glyphs/**
+      - src/archive-font-patcher-readme.me
+      - bin/scripts/archive-font-patcher.sh
+      - bin/scripts/name_parser
+      - .github/workflows/release_master_fontpatcher-zip.yaml
+  workflow_dispatch:
+  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    
+    - name: Grab the script and its dependencies from the repo
+      uses: snow-actions/sparse-checkout@v1.2.0
```

Comment:
In other workflows with equivalent step we use `Bhacaz/checkout-files@v2`.
Maybe use that here also or also switch the other ones?
Can you give details? The less actions we use the easier are updates ;)

I guess this one uses patterns while the other one needs concrete files (but I did not check).

Reply:
See my below conversation: the action you use to checkout in the other actions, at least I think that’s the checkout action I am thinking of, messed with/reset permissions by grabbing files using GitHub’s HTTPS api. This one uses good ol’ git under the hood. 

I think it would be less “dirty” to use a `git` (the CLI)-based action like this whenever possible (ie., maybe indeed switch other workflows to it?), and you can see in my action runs it still is pretty fast for this big clunker of a repo

That all was the reason I picked another action, anyway — you (understandably) didn’t like the manual git command twiddling I’d started with, but I really didn’t want to clone even the entire worktree when we are just using a small section of it.